index.html

```diff
@@ -1,21 +1,520 @@
+              text-center
+              mt-4
+              md:pt-4 md:pr-6.125 md:pb-3.5 md:pl-6.625 md:mt-6
+              w-full
+              md:w-max
+            "
+          >
+            Schedule a Demo
+          </button>
+        </div>
+        <p
+          class="
+            body-copy-1
+            mt-6
+            pb-20
+            text-center
+            font-normal
+            text-secondary-light-san-juan-blue
+          "
+        >
+          Have any questions? <span class="font-bold">Contact Us</span>
+        </p>
+      </div>
+    </div>
+    <div
+      class="
+        flex flex-col
+        justify-center
+        laptop-large:gap-32 laptop-large:flex-row-reverse
+        px-6
+        laptop-large:px-0
+        items-center
+        bg-secondary-mirage-blue
+      "
+    >
+      <div
+        class="
+          grid grid-cols-2
+          md:grid-cols-3
+          gap-12
+          pt-20
+          laptop-large:pt-0
+          px-6
+        "
+      >
+        <img
+          class="fill-current bg-secondary-mirage-blue text-white"
+          src="./styles/assets/shared/desktop/tesla-white.svg"
+          alt=""
+        />
+        <img src="./styles/assets/shared/desktop/microsoft-white.svg" alt="" />
+        <img
+          src="./styles/assets/shared/desktop/hewlett-packard-white.svg"
+          alt=""
+        />
+        <img src="./styles/assets/shared/desktop/oracle-white.svg" alt="" />
+        <div class="flex justify-center items-center">
+          <img src="./styles/assets/shared/desktop/google-white.svg" alt="" />
+        </div>
+        <img src="./styles/assets/shared/desktop/nvidia-white.svg" alt="" />
+      </div>
+      <div
+        class="
+          flex flex-col
+          justify-center
+          items-center
+          laptop-large:items-start
+          max-w-md
+          pt-14
+          md:pt-16
+        "
+      >
+        <p
+          class="
+            heading-3-big
+            text-center
+            md:text-5xl
+            font-normal
+            text-primary-white
+          "
+        >
+          Who we work with
+        </p>
+        <p
+          class="
+            body-copy-1
+            pt-4
+            pb-8
+            text-center
+            laptop-large:text-left
+            text-primary-white
+            opacity-70
+          "
+        >
+          Today, millions of people around the world have successfully connected
+          their accounts to apps they love using our API. We provide developers
+          with the tools they need to create easy and accessible experiences for
+          their users.
+        </p>
+        <button class="btn btn--secondary text-center mb-20">About Us</button>
+      </div>
+    </div>
+    <div
+      class="
+        flex flex-col
+        laptop-large:flex-row
+        justify-center
+        items-center
+        px-6
+        md:px-24
+        pt-20
+        pb-10
+      "
+    >
+      <img
```

Comment:
This `svg` is not showing code indentation.

expected visual
<img width="300" alt="Screen Shot 2021-11-22 at 8 14 33 PM" src="https://user-images.githubusercontent.com/26441464/142957869-d3abe5f7-81eb-48ec-adc7-522a919d049e.png">


current visual
<img width="375" alt="Screen Shot 2021-11-22 at 8 12 28 PM" src="https://user-images.githubusercontent.com/26441464/142957774-a50cadfd-5596-41a4-be49-d03b8ad6827f.png">

Reply:
Closing because we can't find a reasonable solution.